packages/dx-react-grid-bootstrap3/src/plugins/table-column-visibility.jsx

```diff
@@ -3,21 +3,32 @@ import PropTypes from 'prop-types';
 import { TableColumnVisibility as TableColumnVisibilityBase } from '@devexpress/dx-react-grid';
 import { EmptyMessage } from '../templates/empty-message';
 
-export const TableColumnVisibility = ({ emptyMessageText, ...restProps }) => {
-  const emptyMessageTemplate = () => <EmptyMessage text={emptyMessageText} />;
+const emptyMessageTemplate = props => <EmptyMessage {...props} />;
 
-  return (
-    <TableColumnVisibilityBase
-      emptyMessageTemplate={emptyMessageTemplate}
-      {...restProps}
-    />
-  );
+const defaultMessages = {
+  noColumns: 'Nothing to show',
```

Comment:
Will these messages be localized also?

Reply:
For now, we decided not to show column chooser in the localization demos to keep them simple. Of course, a user can localize this message in his app.